2022-05-metafacture-workshop/index.html

```diff
@@ -0,0 +1,939 @@
+			</section>
+
+			<section>
+				<h1>Dokumentation</h1>
+				<p><span class="fragment">Was für Module gibt es? Was machen die?</span></p>
+				<span class="fragment"><pre><code class="bash" data-trim>
+$ flux.sh # oder flux.bat auf Windows
+				</code></pre></span>
+				<p><span class="fragment"><a href="https://github.com/metafacture/metafacture-documentation/blob/master/flux-commands.md">
+				https://github.com/metafacture/metafacture-documentation/blob/master/flux-commands.md</a></span></p>
+				<p><span class="fragment">z.B. statt JSON als "Formeta" ausgeben, formatiert</span></p>
```

Comment:
Abschnitt würde ich vor die Workflow-Erklärung ziehen.

Reply:
Die Idee ist dass die ja erstmal einen fertigen Workflow vorgesetzt bekommen. Wenn es dann daran geht, da andere Module und Optionen zu nutzen, muss man in die Doku gucken, dazu direkt die Aufgabe was nachzusehen (Formeta, formatiert).